app/assets/src/components/visualizations/GenomeViz.js

```diff
@@ -71,23 +71,18 @@ export default class GenomeViz {
     return [min(mins), max(maxs)];
   };
 
-  // Find the bar that is closest to hoverX, within hoverBuffer.
-  onMouseMove = () => {
-    if (this.sortedEndpoints.length == 0 || !this.endpointToDataIndex) {
-      return;
-    }
-
-    const hoverX = mouse(this.svg.node())[0];
+  // Find the bar that is closest to mouseX, within hoverBuffer.
+  getDataIndexForMouseX = mouseX => {
```

Comment:
is there any difference between clientX and mouseX?

Reply:
mouseX is actually a point with respect to the svg container.
clientX is with respect to the page viewport.

I changed "mouseX" to "svgX" and added a comment so it's clearer.